src/scippnexus/nxtransformations.py

```diff
@@ -117,7 +119,12 @@ def make_transformation(
                     depends_on_to_relative_path(depends_on, self._obj.parent.name)
                 )
             return transform
-        except (sc.DimensionError, sc.UnitError, TransformationError):
+        except (sc.DimensionError, sc.UnitError, TransformationError) as e:
+            msg = (
+                f"Failed to load {self.name} as {NXtransformations.__name__}: {e} "
+                "Falling back to returning non transformed value."
```

Comment:
`NXtransformations` is the parent group and loading that did not fail.

```suggestion
                f"Failed to convert {self.name} into a transformation: {e} "
                "Falling back to returning non underlying value."
```